src/main/java/frc/robot/commands/ConveyorCommands/ConveyInwardCommand.java

```diff
@@ -13,6 +15,12 @@ public ConveyInwardCommand(ConveyorSubsystem conveyor){
     @Override
     public void execute(){
         conveyor.conveyInward();
+        isConveying = true;
+    }
+
+    @Override
+    public boolean isFinished(){
+        return isConveying;
```

Comment:
What is the finishing logic that you want from this command? This probably isn't the `isFinished` condition that you want. `execute` sets `isConveying` to `true`, so this command immediately finishes.

If you do want the command to finish immediately, you should just return `true` from `isFinished` directly.

Reply:
updated :)